app/src/main/java/eu/darken/capod/main/ui/widget/WidgetProvider.kt

```diff
@@ -95,6 +96,14 @@ class WidgetProvider : AppWidgetProvider() {
         }
     }
 
+    private fun getCellsForSize(size: Int): Int {
+        var n = 2
+        while (70 * n - 30 < size) {
```

Comment:
Can you explain the algorithm here? Where does the 70 and 30 values come from?

Reply:
I've added some comments explaining this segment of code. Hopefully it's enough.